custom_attributes.go

```diff
@@ -169,3 +170,85 @@ func (s *CustomAttributesService) deleteCustomAttribute(resource string, id int,
 	}
 	return s.client.Do(req, nil)
 }
+
+// ListUsersWithCustomAttributes lists users that have the specified custom attributes.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/users.html#for-admins
+func (s *CustomAttributesService) ListUsersWithCustomAttributes(customAttributes []CustomAttribute, options ...OptionFunc) ([]*User, *Response, error) {
+	return s.listUsersWithCustomAttributes(customAttributes, options...)
+}
+
+func (s *CustomAttributesService) listUsersWithCustomAttributes(customAttributes []CustomAttribute, options ...OptionFunc) ([]*User, *Response, error) {
```

Comment:
Can you maybe share the idea behind adding both a public and private method for the same action? Especially since they seem to have a 1:1 relation, I wonder if we should take this approach here?